resources/views/cities/index.blade.php

```diff
@@ -0,0 +1,167 @@
+            <div class="sm:flex-auto">
+                <p class="mt-2 text-sm text-gray-700">Manage the cities where the airlines and their flights operate.</p>
+            </div>
+        </div>
+        <div class="my-8 flow-root">
+            <div class="-mx-4 -my-2 overflow-x-auto sm:-mx-6 lg:-mx-8">
+                <div class="inline-block min-w-full py-2 align-middle sm:px-6 lg:px-8">
+                    <div class="overflow-hidden shadow ring-1 ring-black ring-opacity-5 sm:rounded-lg">
+                        <table class="min-w-full divide-y divide-gray-300">
+                            <thead class="bg-gray-50">
+                                <tr>
+                                    <th scope="col"
+                                        class="py-3.5 pl-4 pr-3 text-left text-sm font-semibold text-gray-900 sm:pl-6" onClick="setSorting('id')">
+                                        Id</th>
+                                    <th scope="col"
+                                        class="px-3 py-3.5 text-left text-sm font-semibold text-gray-900" onClick="setSorting('name')">Name</th>
+                                    <th scope="col"
+                                        class="px-3 py-3.5 text-left text-sm font-semibold text-gray-900">Incoming
+                                        flights</th>
+                                    <th scope="col"
+                                        class="px-3 py-3.5 text-left text-sm font-semibold text-gray-900">Outgoing
+                                        flights</th>
+                                    <th scope="col" class="relative py-3.5 pl-3 pr-4 sm:pr-6">
+                                        <span class="sr-only">Edit/Delete</span>
+                                    </th>
+                                </tr>
+                            </thead>
+                            <tbody class="divide-y divide-gray-200 bg-white">
+                                <!-- Data is fetched using AJAX -->
+                            </tbody>
+                        </table>
+                        <div id="pagination" class="bg-gray-50">
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="flex justify-end">
+            <div class="flex-auto">
+                <x-input class="w-full" placeholder="City name..."></x-input>
+            </div>
+            <div class="mt-4 sm:ml-2 sm:mt-0 flex-auto">
+                <x-button onclick="createCity()">Add city</x-button>
+            </div>
+        </div>
+    </div>
+</x-layout>
+
+<script>
+    let sort = null;
+    let order = null;
+    function fetchCities(page) {
+        url = '/api/cities?page=' + page;
+        if (sort) {
+            url += '&sort=' + sort;
+        }
+        if (order) {
+            url += '&order=' + order;
+        }
+        $.ajax({
+            type: 'GET',
+            url: url,
```

Comment:
you could use data: to add params to your query instead of concatenating strings to your url